src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheckTest.java

```diff
@@ -186,9 +186,8 @@ public void testJavadocCommentAfterPackageWithImports() throws Exception {
         final String[] expected = {
             "2:1: " + getCheckMessage(MSG_SHOULD_BE_SEPARATED, "/*"),
         };
-        verify(checkConfig,
-                getPath("InputEmptyLineSeparatorJavadocCommentAfterPackage.java"),
-                expected);
+        execute(checkConfig,
+                getPath("InputEmptyLineSeparatorJavadocCommentAfterPackage.java"));
```

Comment:
Remove "expected" variable

Reply:
Changes done

Reply:
You need to convert to new method only places where we expect exception.

All other should be converted to method that use inlined config in Input files.

Reply:
I believe that we should not use `execute `here as I am getting PMD error and If I am using `veriftWithInlineConfigParser` and changing the `config `in input files then all tests are passing.

But after changing the input files, I am getting code coverage issue.

Reply:
update test to not use "verify"  and use verifyWithInlineConfigParser